example-context/example-service/example-command/example-command-handler/src/main/java/uk/gov/justice/services/example/cakeshop/command/handler/AddRecipeCommandHandler.java

```diff
@@ -44,7 +45,7 @@
     Enveloper enveloper;
 
     @Handles("cakeshop.add-recipe")
-    public void addRecipe(final JsonEnvelope command) throws EventStreamException {
+    public void addRecipe(final JsonEnvelope command) throws EventStreamException, InvalidSequenceIdException {
```

Comment:
What is throwing this InvalidSequenceIdException? Do we need to make the developers catch another exception that they don't really care about?